schema/schema.json

```diff
@@ -139,6 +139,31 @@
                 "noNewPrivileges": {
                     "id": "https://opencontainers.org/schema/bundle/process/linux/noNewPrivileges",
                     "type": "boolean"
+                },
+                "rlimits": {
+                    "id": "https://opencontainers.org/schema/bundle/linux/rlimits",
+                    "type": "array",
+                    "items": [
+                        {
```

Comment:
On Wed, Apr 06, 2016 at 07:43:10PM -0700, Vincent Batts wrote:
> > @@ -139,6 +139,31 @@
> >                  "noNewPrivileges": {
> >                      "id": "https://opencontainers.org/schema/bundle/process/linux/noNewPrivileges",
> >                      "type": "boolean"
> > +                },
> > +                "rlimits": {
> > +                    "id": "https://opencontainers.org/schema/bundle/linux/rlimits",
> > +                    "type": "array",
> > +                    "items": [
> > +                        {
> 
> fair. though it is not strict. I can change it, but it works as is.

Does it work with zero or multiple entries as is?  I'd expect a schema
array with one object to only validate the first entry in the config
array.